AppCenterReactNativeShared/android/src/main/java/com/microsoft/appcenter/reactnative/shared/AppCenterReactNativeShared.java

```diff
@@ -84,16 +68,19 @@ public static String getAppSecret() {
                 configStream.close();
                 String jsonContents = new String(buffer, "UTF-8");
                 JSONObject json = new JSONObject(jsonContents);
-                sAppSecret = json.getString(APP_SECRET_KEY);
+                sAppSecret = json.optString(APP_SECRET_KEY);
```

Comment:
To align with iOS where missing key in plist works, optString returns empty if key missing